### ISSUE TITLE
Redesign Resident Permit Parking Program pamphlet

### ISSUE THREAD

**Issue by carolinestj** [CityOfBoston/design-team-workflow#556]

**Comment by carolinestj** [comment]
[DOC121817-12182017164908.pdf](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/57fcece44095e71300552840/125516c66e82c728ace21e0d46ccbbb972d0d2f1debef212f60ca8c6791770227f50fa7b28c7af6aaf50740b0a0c0abe4b4a5a43aff98e2ef3b1397888090bb8d36c5f76976815aaeb8b0da313f871d51bdaebe5c01982fa3737e682854e3b1f061e8aff5931a2984fce4abb.pdf)

https://docs.google.com/document/d/1XkTNnzVFzoBxdBPLvSs0ndFJ7Xev4IZKFVMfHOa46UA/edit

**Comment by carolinestj** [comment]
Switch x & y axis for graphs 

Change color order for Back Bay & seaport parking graph, so the highest concentration has red color and then down the color spectrum as the concentration reduces

new graph showing cars parked (waiting on content for that) 

check text for double sentences and make sure things are bolded where they need to be bolded

Add New titles for Back Bay & Seaport parking graphs  

new text should be done on the 8th of january 
final report tentatively due Jan 15th
